torch/_torch_docs.py

```diff
@@ -2360,6 +2360,10 @@ def merge_dicts(*dicts):
 corresponding to the polar coordinates with absolute value :attr:`abs` and angle
 :attr:`angle`.
 
+.. note::
+    This implementation follows C++'s `std::polar <https://en.cppreference.com/w/cpp/numeric/complex/polar>`_
```

Comment:
"follows" is vague - how is torch.polar like C++'s std::polar?

Reply:
(does torch.polar always call std::polar?)

Reply:
we do follow the same logic as `std::polar`, however we don't call into `std::polar`. 
https://github.com/pytorch/pytorch/blob/a9b0a921d592b328e7e80a436ef065dadda5f01b/aten/src/ATen/native/cpu/ComplexKernel.cpp#L18-L24 I don't see a reason why we shouldn't though!

Reply:
I am updating the description to say that this function follows the exact logic of `std::polar` and that the input is unvalidated (i.e. it will still perform that computation as described even if the `abs` input tensor contains negative values`.